sources/web/datalab/static/appbar.html

```diff
@@ -13,6 +13,11 @@
   </span>
   <div class="btn-toolbar pull-right">
     <div class="btn-group">
+      <button id="ungitButton" title="Open ungit here" class="toolbar-btn">
```

Comment:
One comment I heard from Nikhil: Should this button (and ungit) only be there when running on a VM?